theano/tensor/opt.py

```diff
@@ -2317,7 +2319,8 @@ def local_upcast_elemwise_constant_inputs(node):
                 else:
                     try:
                         # works only for scalars
-                        cval_i = get_scalar_constant_value(i, elemwise=False)
+                        cval_i = get_scalar_constant_value(i, elemwise=False,
```

Comment:
here you can remove elemwise=False. when we process only constant, this parameter isn't used.